src/lib/assets/webhooks.ts

```diff
@@ -21,7 +21,7 @@ const peprIgnoreLabel: V1LabelSelectorRequirement = {
 const peprIgnoreNamespaces: string[] = ["kube-system", "pepr-system"];
 
 export async function generateWebhookRules(assets: Assets, isMutateWebhook: boolean) {
-  const { config, capabilities } = assets;
+  const { config, capabilities = [] } = assets;
```

Comment:
This seems due to a non-null assertion [here](https://github.com/defenseunicorns/pepr/blob/ec6922a27a3d464a83c6347c43e6554d4a746d84/src/lib/assets/index.ts#L28). Can we initialize `capabilities` to `[]` in that file so that we confine any initialization quirks to one spot?

Reply:
✅ Done